### PR TITLE
add galician translation gl_ES

### PR DESCRIPTION
**Pull request by ninjum**
added translation into Galician

### CODE REVIEW

File: data/language/gl_ES.txt [hunk 1]
```diff
@@ -0,0 +1,4579 @@
...
+STR_6653    :Todas as fontes mostradas
+STR_6654    :Amosando {POP16}{UINT16} fontes
+STR_6655    :Só ‘{POP16}{STRINGID}’
+STR_6656    :Elimina todos os valados do parque
+STR_6657    :Terra Sen Propietario
+STR_6658    :Establecer que a terra non sexa propiedade do parque nin estea dispoñible para a compra
+STR_6659    :Os invitados ignoran os prezos
+STR_6660    :Os invitados ignorarán o prezo das atraccións e dos postos
+
+##############
```
Comment (Gymnasiast) [inline data/language/gl_ES.txt:3734]: We moved the scenario translations elsewhere (as of a few days). Could you remove them from this PR? After adding Galician to the game, we will set up everything for you so you can translate all of the objects and scenarios.

Also, are you prepared to maintain this translation for some time to come?

Comment (ninjum) [reply data/language/gl_ES.txt:3734]: yes, the idea is to maintain it as time permits.

Comment (Gymnasiast) [reply data/language/gl_ES.txt:3734]: Nice to hear! Could you remove the scenario translations from the PR?  I’ll take care of the process from there.

Comment (ninjum) [reply data/language/gl_ES.txt]: removed the scenarios, if something else is missing let me know and it will be done

**Comment by Gymnasiast** [comment]
Can you rename the file to `gl-ES.txt` (replacing the underscore with a dash)?

**Comment by ninjum** [comment]
Changed underscore to dash.

**Pull request merged**